plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlClient.java

```diff
@@ -285,20 +285,18 @@ public void testConvertComparison()
             switch (operator) {
                 case EQUAL:
                 case NOT_EQUAL:
-                    assertThat(converted).isPresent();
-                    assertThat(converted.get().expression()).isEqualTo(format("(\"c_bigint\") %s (?)", operator.getValue()));
-                    assertThat(converted.get().parameters()).isEqualTo(List.of(new QueryParameter(BIGINT, Optional.of(42L))));
-                    return;
```

Comment:
Good catch. This is left over from https://github.com/trinodb/trino/commit/b561a3399aef3b9445da31812d2dcda75189bbf6 (https://github.com/trinodb/trino/pull/19766)
It's not the first time we realize the `return` was inside the test code, and test cases are not executing.
Can you maybe review that commit?

Reply:
I can look into it.